tests/api/utils/test_scheduler.py

```diff
@@ -163,17 +170,15 @@ async def test_invoke_schedule(
 
 
 @pytest.mark.asyncio
-async def test_create_schedule_mlrun_function(
-    db: Session,
-    scheduler: Scheduler,
-    k8s_secrets_mock: tests.api.conftest.K8sSecretsMock,
-):
-    now = datetime.now()
-    now_plus_1_second = now + timedelta(seconds=1)
-    now_plus_2_second = now + timedelta(seconds=2)
+async def test_create_schedule_mlrun_function(db: Session, scheduler: Scheduler):
```

Comment:
Looks like you removed the `k8s_secrets_mock` from the test function signature, if the tests passed it's probably not needed, but if it doesn't, add it back

Reply:
It's doesn't used in the function.